.changeset/cold-beds-sort.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sumup/circuit-ui': minor
+---
+
+Added the `inputOutline` style mixin. It can be used to communicates to the user that an element is hovered, focused, or active; including for the disabled, invalid and warning states.
```

Comment:
```suggestion
Added the `inputOutline` style mixin. It can be used to communicate to the user that an element is hovered, focused, or active in the disabled, invalid, and warning states.
```